lib/account/account-service.js

```diff
@@ -0,0 +1,99 @@
+"use strict";
+
+/**
+ * What we initially send back to the client so they can start to initiate
+ * the complete procedure and save information.
+ *
+ * @typedef {accountService~initiateResult}
+ * @param {string} accountId
+ * @param {string} email
+ * @param {string} mfa
```

Comment:
Should rename this to be more clear.   This is the MFA seed or key, not a MFA generated value.